src/App.js

```diff
@@ -1,8 +1,170 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import Review from './Review';
+import Modal from 'react-modal';
+import { ModalForm } from './ModalForm';
+import { Box, Button, Footer, Grommet, Header, Heading, Main } from 'grommet';
+import { AddCircle } from 'grommet-icons';
+
+// https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Statements/export
+
 function App() {
+  const [reviews, setReviews] = useState([])
+  const [isModalOpened, setIsModalOpened] = useState(false)
+  const initialFormValues = {
+    id: null,
+    title: '',
+    score: 1,
+    body: '',
+    reviewer: ''
+  }
+  const [formValues, setFormValues] = useState(initialFormValues)
+
+  const upper = {
+    position: 'fixed',
+    right: '50px',
+    bottom: '50px',
+    transition: '1s',
+    opacity: '0.7'
+    }
+
+  const customStyles = {
+    content : {
+      top                   : '50%',
+      left                  : '50%',
+      right                 : 'auto',
+      bottom                : 'auto',
+      marginRight           : '-50%',
+      transform             : 'translate(-50%, -50%)'
+    }
+  };
+
+  const theme = {
+    global: {
+      font: {
+        family: 'Roboto',
+        size: '18px',
+        height: '20px',
+      },
+    },
+  };
+
+  async function getReviews() {
+    const response = await axios.get('https://bookreview-ten.vercel.app/api/reviews');
+    return response.data
+  }
+
+  useEffect(() => {
+    async function initializeReviews() {
+      const data = await getReviews();
+      setReviews(data)
+    }
+
+    initializeReviews()
```

Comment:
debug 用に関数名を付けたいだけなら、すぐ実行しちゃっていいですよ？

```suggestion
    (async function initializeReviews() {
      const data = await getReviews();
      setReviews(data)
    })()
```